app/models/season.rb

```diff
@@ -39,7 +39,7 @@ def projects_proposable?
     end
 
     def all_years
-      pluck(:name)
+      where("name <= ?", Date.today.year.to_s).order(:name).pluck(:name)
```

Comment:
Until Dec  31st of the current year, it will return all the years that have teams. As it should! So that's great. 
But on Jan 1st (and all the way until we actually have teams _in_ the _upcoming_ season), it will incorrectly show that upcoming as a choice. 
For example: the current season is '2018'. Next season is "2019". On 2019 Jan 1st this method will return 2019 as well, because `"2019" <= "2019"` is true.  But there are no teams for 2019, so '2019' should not show up in the selection. 

So, your solution works absolutely to include the current year 🎉 . The caveat is that it sometimes _shouldn't_ return the current year.

Are you up to the challenge to try to solve this? 
(Also, this means that the method name may not be the best choice...)
FYI: the readme mentions the Season Switch, that you can use to fake that we are in the 'proposal phase' of a season, and in that phase there are no teams for that season yet.

Reply:
We have something like an `active` season. Out of my head, I'd say that's your best bet. Or at least it should give you the information you need. 👍

Reply:
Thanks!